apps/src/lib/ui/accounts/ManageLinkedAccounts.jsx

```diff
@@ -91,6 +103,13 @@ class ManageLinkedAccounts extends React.Component {
     }
   };
 
+  connectDisabledStatus = authOption => {
```

Comment:
Is it possible to combine `connectDisabledStatus` and `disconnectDisabledStatus` into one `getDisabledStatus` function? In my mind, it seems like it'd be cleaner to have one function, and more consistent with the `OauthConnection` having one `disabledStatus` prop and one function that gets the tooltip regardless of whether it's a connect or disconnect button. But I haven't thought through all the possible implications.